### PR TITLE
feat(mode switcher): integrated mode/mode variant switcher

### PR DESCRIPTION
**Pull request by mJarsater**

### CODE REVIEW

File: src/app/components/accordion/accordion.component.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h2>Icon Position</h2>
 
 <br />
 <h2>Accordion with Primary Mode Variant</h2>
-<tds-accordion mode-variant="primary">
+<tds-accordion>
```
Comment (mJarsater) [inline src/app/components/accordion/accordion.component.html:37]:
```suggestion
<tds-accordion mode-variant="primary">
```

File: src/app/mode-switcher/mode-switcher.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<tds-toggle (tdsToggle)="toggleMode()">
+    <div slot="label">Mode</div>
+</tds-toggle>
```
Comment (theJohnnyMe) [inline src/app/mode-switcher/mode-switcher.component.html:3]:
```suggestion
<tds-toggle (tdsToggle)="toggleMode()">
    <div slot="label">Enable dark mode</div>
</tds-toggle>
```
Just suggestion...

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,114 +1,124 @@
...
+            </tds-side-menu-user>
+          </button>
+        </tds-side-menu-item>
+    
+        <tds-side-menu-collapse-button slot="sticky-end"></tds-side-menu-collapse-button>
+      </tds-side-menu>
+      <main class="tds-u-h-100 tds-u-w-100">
+        <div class="tds-u-h-100 tds-u-p3">
+          <app-mode-switcher (modeToggle)="handleModeToggle()"></app-mode-switcher>
+          <app-mode-variant-switcher (modeVariantToggle)="handleModeVariantToggle()"></app-mode-variant-switcher>
```
Comment (theJohnnyMe) [inline src/app/app.component.html:114]:
```suggestion
<span class="tds-u-flex tds-u-flex-gap-1 tds-u-flex-dir-col tds-u-pb3">
          <app-mode-switcher (modeToggle)="handleModeToggle()"></app-mode-switcher>
          <app-mode-variant-switcher (modeVariantToggle)="handleModeVariantToggle()"></app-mode-variant-switcher>
          </span>
```

Just to make some spacing between them and after them...

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Pull request merged**